pages/docs/quickstart-forms.mdx

```diff
@@ -12,7 +13,7 @@ You can create your Profile by completing a Form on the BioDrop app.
 
 To do this, you need a GitHub account. If you do not have one yet, you can create one for free with an email address and password.
 
-1. Go to the [BioDrop homepage](https://biodrop.io/)
+1. Go to the <Link href={BASE_WEBSITE_URL} target="_blank" rel="noopener">`BioDrop homepage`</Link>
```

Comment:
I don't think this should be in backtics though because it does look like a link

<img width="320" alt="Screenshot 2023-09-21 at 11 16 18" src="https://github.com/EddieHubCommunity/BioDrop/assets/624760/1300e2aa-c2f6-473e-a527-7ff7fc0fec3e">

Reply:
ah, earlier it was not visible in dark mode, that's why i put them. But now its fine. Fixed.